src/loader/doc_attr.ml

```diff
@@ -139,7 +139,7 @@ let extract_top_comment internal_tags ~classify parent items =
   let rec extract ~classify = function
     | hd :: tl as items -> (
         match classify hd with
-        | Some (`Attribute attr) -> (
+        | Some (`Attribute ({ Parsetree.attr_name = { txt = "ocaml.text" ; _ }; _ } as attr)) -> (
```

Comment:
Matching this is already the responsibility of `parse_attribute`. I think what you mean is to change the `None` case below to continue iterating rather than stopping.

Reply:
Agreed - nice find though!

Reply:
Indeed, i missed that. Thanks !